config/config.php

```diff
@@ -692,7 +692,7 @@
 yUhYjn9bUBv6WmsmPNeesA==',
 			],
 		],
-		'29.0.8.0' => [
+		'29.0.8.1' => [
```

Comment:
should this point to 30.0.1 RCs or 30.0.0 final?

Reply:
beta channel, thus 30.0.1 RC1 atm